contracts/serializer.scrypt

```diff
@@ -48,7 +48,7 @@ contract Reader {
 
```

Comment:
All
this.pos = this.pos + 1;
-> 
this.pos++;


this.pos = this.pos + len;
->
this.pos += len;